gatsby-node.js

```diff
@@ -1,10 +1,19 @@
 /* eslint-env node */
 
+const fs = require('fs')
```

Comment:
not for this PR: this file is clearly getting out of hand - it's close to impossible to review it. We should split into docs and blog is possible at all, move common helpers outside?

Reply:
Agree, I will try to do it in the next PR.